src/project-1.js

```diff
@@ -2,139 +2,262 @@
 
 const multiplyByTen = (num) => {
   // return num after multiplying it by ten
-  // code here
+  const byTen = num * 10;
+  return byTen;
 };
+const numByTen = multiplyByTen (5);
+
+console.log (numByTen);
 
 const subtractFive = (num) => {
   // return num after subtracting five
-  // code here
+  const minusFive = num - 5;
+  return minusFive;
 };
+const numMinusFive = subtractFive (5);
+
+console.log (numMinusFive);
 
 const areSameLength = (str1, str2) => {
   // return true if the two strings have the same length
   // otherwise return false
   // code here
+  if (str1.length === str2.length) {
+    return true; 
+  } else {
+    return false;
+  }
 };
 
+const checkLength = areSameLength ('boat', 'school');
+
+console.log (checkLength);
+
 const areEqual = (x, y) => {
   // return true if x and y are the same
   // otherwise return false
   // code here
+  if (x === y) {
+    return true;
+  } else {
+    return false;
+  }
 };
+const same = areEqual(5, 4); 
+console.log(same);
 
 const lessThanNinety = (num) => {
   // return true if num is less than ninety
   // otherwise return false
   // code here
+  if (num < 90) {
+    return true;
+  } else { 
+    return false;
+  }
 };
+const lessNine = lessThanNinety(45); 
+console.log(lessNine); 
 
 const greaterThanFifty = (num) => {
   // return true if num is greater than fifty
   // otherwise return false
-  // code here
+  // code here 
+  if (num > 50) {
+    return true;
+  } else {
+    return false;
+  }
 };
+const moreFifty = greaterThanFifty(45); 
+console.log(moreFifty);
 
 const add = (x, y) => {
   // add x and y together and return the value
-  // code here
+  // code here 
+  const add = x + y;
```

Comment:
Try not to use the same name as the function for this. Sum would probably be a better name as it describes the value. Otherwise, you could also just do this: return x + y;